src/macros.rs

```diff
@@ -354,39 +354,48 @@ macro_rules! BITFIELD {
         )+}
     }
 }
+
 #[macro_export]
+#[allow(non_snake_case)]
 macro_rules! ENUM {
```

Comment:
note: macro names should be in lower-case too. I think a more fitting name here should be `win_enum`